OpenRA.Mods.Common/Widgets/RadarWidget.cs

```diff
@@ -87,6 +87,7 @@ public override void Initialize(WidgetArgs args)
 			foreach (var cell in world.Map.AllCells)
 				UpdateTerrainCell(cell);
 
+			// TODO: Ideally, we need to detach these event handlers when the radar widget is finished with.
```

Comment:
We have `Widget.Removed()` for when a widget is removed (not just hidden) from view.  I'm not 100% sure this is used as a dispose-equivalent, but it may well be.